docker-compose.hobby.yml

```diff
@@ -128,7 +128,7 @@ services:
 
     asyncmigrationscheck:
         <<: *worker
-        command: python manage.py run_async_migrations --complete-noop-migrations && python manage.py run_async_migrations --check
```

Comment:
I was tagged but TBH I don't know what this does 🤔

Reply:
Right I'll remove you